config/webpack.prod.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = Merge.smart(commonConfig, {
         test: /\.(js|jsx)$/,
         include: [
           path.resolve(__dirname, '../src'),
+          path.resolve(__dirname, 'node_modules/camelcase-keys'),
```

Comment:
DRYing this up would be nice, but I couldn't decide how to do it and it didn't seem THAT important yet.